backend/tests/integration/orcid/test_orcid_cli_import_legacy_tokens.py

```diff
@@ -191,22 +236,27 @@ def test_import_multiple_orcid_tokens_no_user_exists(
     assert_db_has_n_legacy_tokens(0, SAMPLE_USER_2)
 
     # Migrate
-    import_legacy_orcid_tokens()
+    result = app_cli_runner.invoke(import_legacy_orcid_tokens)
+
+    # both authors are not found
+    output = result.output.split('\n')
+    assert output.count('No row was found for one()') == 2
```

Comment:
do we care about the output? just checking the app if the tokens are there and the queue is empty is not enough?

Reply:
I didn't want to change the test logic, that's why I kept them. Some tests are actually focusing on the code throwing exceptions.